js/viz/docs/docObjectsDxChart.js

```diff
@@ -94,3 +106,23 @@ var chartPointObject = {
     */
     getBoundingRect: function() { }
 };
+
+/**
+* @name chartAxisObject
+* @publicName Axis
```

Comment:
Please, don't use publicName for this object.

Reply:
Done.